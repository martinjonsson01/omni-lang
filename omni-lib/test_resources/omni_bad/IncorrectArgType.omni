module IncorrectArgType()

function(x: Int, y: Int) : Unit = ()

main() : Unit = function(1, ())