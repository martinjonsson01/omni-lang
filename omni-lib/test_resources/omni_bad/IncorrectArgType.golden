error:
expected type Int
but
'EUnit (Src (Loc {sourceFile = "test_resources\\omni_bad\\IncorrectArgType.omni", range = SrcRange {rangeFrom = SrcPos {posLine = 5, posColumn = 29}, rangeTo = SrcPos {posLine = 5, posColumn = 31}}}))'
has type
Unit
     